Automattic/jetpack
# Admin Page - Remove the WordPress Apps banner

Fixes #11937

#### Changes proposed in this Pull Request:


* Removes the app component
* Moves the former apps-card/style.css to be themes-promo-card/style.css as this card was reusing some styles from the AppCard component.
* Removes the dismissal setting behaviour from endpoint.

#### Testing instructions:

* Checkout this branch or [launch a JN site with this branch](https://jurassic.ninja/create?jetpack-beta&branch=remove/apps-card-from-admin-page&wp-debug-log).
* Visit the admin page for Jetpack
* Confirm you no longer see the Cards App
* Visit the Plans tab, emulate a free plan and confirm the Themes Promo card looks well

#### Proposed changelog entry for your changes:


* Removes the WordPress.com Apps banner from Jetpack settings page
